static/templates/admin_user_list.handlebars

```diff
@@ -2,6 +2,9 @@
 <tr class="user_row{{#unless is_active}} deactivated_user{{/unless}}" data-user-id="{{user_id}}" data-email="{{email}}">
     <td>
         <span class="user_name">{{full_name}}</span>
+        {{#if is_admin}}
+        &nbsp;<i class="fa fa-bolt" aria-hidden="true"></i>
+        {{/if}}
```

Comment:
This deserves a label.  The ARIA spec is pretty simple to understand: If the icon conveys information that isn't otherwise available in the text, it needs a title + label; if it doesn't (e.g. because it's just a decoration next to text explaining what it does), then it should have `aria-hidden`

```
-        &nbsp;<i class="fa fa-bolt" aria-hidden="true"></i>
+        &nbsp;<i class="fa fa-bolt" title="{{t 'Organization administrator' }}" aria-label="{{t 'Organization administrator' }}"></i>
```

Also, if you toggle whether a user is an admin, this block should be rerendered; probably will require some new `server_events_dispatch` code.

Reply:
(Also, this commit should swap order with the second commit, rather than squashing the two, since this change would be an improvement even without moving the buttons)